CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## [Unreleased] - YYYY-MM-DD
 
 ### Added
+- Spark Estimator: Add support for custom data loaders in KerasEstimator.
 
```

Comment:
I would also add another line mentioning the main contribution here:
```suggestion
Spark Estimator: Add NVTabular data loader for KerasEstimator. ([#3603](https://github.com/horovod/horovod/pull/3603))
```
Please add that link to the PR also to above CHANGELOG entry.